### DISCUSSION TITLE
Purpose of null weather value in gridpoint data?

### DISCUSSION THREAD

**Discussion by mbg206** [weather-gov/api#668] [Q&A]
I notice that in the weather property of the raw gridpoint data (`/gridpoints/{wfo}/{x},{y}`), there are some objects where the `weather` and other properties are set to null. Is there a meaning to these null weather objects?
![image](https://github.com/weather-gov/api/assets/66435380/6ea615ea-32da-45c1-bdab-ab90a2f1c519)

**Comment by sullynole** [comment]
A null value indicates there is no data in the grid for that layer and time period.
